code/modules/chemistry/Reagents-Base.dm

```diff
@@ -251,6 +251,14 @@
 				H.contract_disease(/datum/ailment/malady/heartdisease,null,null,1)
 			..()
 
+	handle_addiction(var/mob/M, var/rate, var/addProb)
+		if (isliving(M))
+			var/mob/living/H = M
+			if (isalcoholresistant(H))
+				addProb /= 4
+				rate /= 4
+		..()
```

Comment:
```suggestion
		..(M, rate, addProb)
```
I *think* DM will handle the modified argument fine without this, buut it's much clearer what you're doing when it's written like this.